apps/tools/.env

```
@@ -0,0 +1,64 @@
+
+#Crisp marketplace key & identifier to use crisp api
+CRISP_TOKEN_IDENTIFIER=1
+CRISP_TOKEN_KEY=1
+
+# Token d'authentification Notion et identifiants pour la création de bugs
+NOTION_TOKEN=1
+NOTION_BUG_SUPPORT_DATABASE_ID=1
+NOTION_BUG_TEMPLATE_ID=1
+NOTION_BUG_DATABASE_ID=1
+NOTION_BUG_EPIC_ID=1
+NOTION_BUG_TEMPLATE_ID=1
```

dotenv-linter (4.0.0)

[warning] [DuplicatedKey] The NOTION_BUG_TEMPLATE_ID key is duplicated

(DuplicatedKey)